pygments/lexers/dns.py

```diff
@@ -73,23 +73,28 @@ class DnsZoneLexer(RegexLexer):
         'values': [
             (r'\n', Whitespace, "#pop"),
             (r'\(', Punctuation, 'nested'),
-            include('simple-values'),
+            include('simple-value'),
         ],
         # Parsing nested values (...):
         'nested': [
             (r'\)', Punctuation, "#pop"),
-            include('simple-values'),
+            include('multiple-simple-values'),
         ],
         # Parsing values:
-        'simple-values': [
-            (r'(;.*)(\n)', bygroups(Comment.Single, Whitespace)),
+        'simple-value': [
+            (r'(;.*)', bygroups(Comment.Single)),
             (r'[ \t]+', Whitespace),
             (r"@\b", Operator),
             ('"', String, 'string'),
             (r'[0-9]+[smhdw]?$', Number.Integer),
             (r'([0-9]+[smhdw]?)([ \t]+)', bygroups(Number.Integer, Whitespace)),
             (r'\S+', Literal),
         ],
+        'multiple-simple-values': [
+            include('simple-value'),
+            (r'[\n]+', Whitespace),
+            (r'[ \t]+', Whitespace),
```

Comment:
Aren't these two just `\s+`?

Also, `simple-value` already includes a rule for `[ \t]+`.

Reply:
@birkenfeld is there anything else I can do to help getting this merged?